core/src/apps/cardano/layout.py

```diff
@@ -42,6 +51,7 @@
     CardanoCertificateType.STAKE_REGISTRATION: "Stake key registration",
     CardanoCertificateType.STAKE_DEREGISTRATION: "Stake key deregistration",
     CardanoCertificateType.STAKE_DELEGATION: "Stake delegation",
+    CardanoCertificateType.STAKEPOOL_REGISTRATION: "Stakepool registration",
```

Comment:
officially, it is "Stake pool"

Reply:
in ledger specs: https://hydra.iohk.io/build/4339394/download/1/ledger-spec.pdf I could find both stakepool and stake pool, what is the source of truth here?

Reply:
I would go with docs directed at users, e.g. https://cardano.org/stake-pool-operation/.

Reply:
refactored